meta-balena-common/recipes-core/balena-rollback/files/rollback-health

```diff
@@ -45,6 +45,13 @@ run_hooks_from_inactive () {
 	# Allow old OS hooks to access efivars
 	if [ -d /sys/firmware/efi/efivars ]; then
 		mount -t efivarfs efivarfs "${old_rootfs}/sys/firmware/efi/efivars" || true
+
+		# mount securityfs on EFI systems, used for secure boot
+		mount none -t securityfs "${old_rootfs}/sys/kernel/security"
```

Comment:
hey @jakogut what's the use of this securityfs? I might not be up to date with the latest work on this.

Reply:
@alexgg The securityfs contains the binary file that stores TPM event log information. We use the event log to improve compatibility on machines that have unexpected PCR 7 measurements, as well as to discern whether or not a machine measures EFI binaries into PCR 7.